test/multiversion/wt_multiversion.sh

```diff
@@ -5,7 +5,7 @@ last_stable_dir=wiredtiger_${last_stable}/
 last_stable_branch=mongodb-${last_stable}
 
 function setup_last_stable {
-    git clone git@github.com:wiredtiger/wiredtiger.git ${last_stable_dir}
```

Comment:
I don't see an evergreen config that calls this file. Do you know where it's executed from?

Reply:
You could be right. Not sure if anyone uses this script locally. This change is optional with the motivation of aligning the git clone method with other places.